homeassistant/components/enphase_envoy/sensor.py

```diff
@@ -412,7 +390,7 @@ def native_value(self) -> int | None:
         return self.entity_description.value_fn(system_consumption)
 
 
-class EnvoyInverterEntity(EnvoyBaseEntity, SensorEntity):
+class EnvoyInverterEntity(EnvoySensorBaseEntity, SensorEntity):
     """Envoy inverter entity."""
 
     _attr_icon = ICON
```

Comment:
Line 397 still has `_attr_has_entity_name = True`

Reply:
Everything uses it now so set it in the base entity

20e5aef90ba948b3ea5133e77c5f19705ea761da